independent-publisher-connectors/Connpass/apiDefinition.swagger.json

```diff
@@ -0,0 +1,322 @@
+                        "title": "Number of alternates"
+                      },
+                      "updated_at": {
+                        "type": "string",
+                        "description": "updated_at",
+                        "title": "Update date"
+                      },
+                      "owner_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "owner_id",
+                        "title": "Admin ID"
+                      },
+                      "owner_nickname": {
+                        "type": "string",
+                        "description": "owner_nickname",
+                        "title": "Admin Nickname"
+                      },
+                      "owner_display_name": {
+                        "type": "string",
+                        "description": "owner_display_name",
+                        "title": "Admin display name"
+                      },
+                      "place": {
+                        "type": "string",
+                        "description": "place",
+                        "title": "Place of the event"
+                      },
+                      "address": {
+                        "type": "string",
+                        "description": "address",
+                        "title": "Location"
+                      },
+                      "lat": {
+                        "type": "string",
+                        "description": "lat",
+                        "title": "Latitude of the venue"
+                      },
+                      "lon": {
+                        "type": "string",
+                        "description": "lon",
+                        "title": "Longitude of the venue"
+                      },
+                      "series": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "id",
+                            "title": "Group ID"
+                          },
+                          "title": {
+                            "type": "string",
+                            "description": "title",
+                            "title": "Group title"
+                          },
+                          "url": {
+                            "type": "string",
+                            "description": "url",
+                            "title": "Group URL"
+                          }
+                        },
+                        "description": "series"
+                      }
+                    }
+                  },
+                  "description": "events"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get_event",
+        "operationId": "Get_event",

```

Comment:
@Miyake-Mito please use pascal case for operationId properties. You can [refer to coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) for more information and examples.

Reply:
Thank you for reviewing this.
I have applied pascal case to the operationId properties.